src/components/text-input/TextInput.js

```diff
@@ -46,29 +45,28 @@ const TextInput = ({
     labelTooltip,
     ...rest
 }: Props) => {
-    const classes = classNames(className, 'text-input-container', {
-        'show-error': !!error,
+    const classes = classNames(className, 'bdl-TextInput-container', {
+        'bdl-hasError': !!error,
     });
 
     return (
         <div className={classes}>
             <Label
+                className="bdl-TextInput-label"
```

Comment:
This PR is using https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md as naming example